networking_ccloud/tools/agent_rpc_caller.py

```diff
@@ -58,10 +60,16 @@ def main():
         topic = cc_const.SWITCH_AGENT_TOPIC_MAP[CONF.agent]
         client = CCFabricSwitchAgentRPCClient(topic)
 
+    try:
+        kwargs = json.loads(CONF.kwargs)
+    except json.decoder.JSONDecodeError as e:
+        print(f"Error: Could not decode json in kwargs: {e}")
+        sys.exit(1)
+
     print(f"Doing RPC call {CONF.method}(*{CONF.args}) with topic {topic}")
```

Comment:
Does this not need the kwargs? Imho, it now shows incomplete information about the call that was being made.